src/20250120_Multi_FebruaryFundingUpdate/FebruaryFundingUpdate.md

```diff
@@ -0,0 +1,146 @@
+
+- Fund upcoming GHO incentive campaigns with FLUID;
+  - Follow up action to this proposal;
+- Fund Merit Phase V;
+  - Swap USDC and USDT to GHO;
+- Reimburse ACI of costs incurred from;
+  - Repaying EURA bad debt;
+  - Administering the PYUSD incentives;
+  - Gas costs related to AIP submissions;
+- Deposit passively held assets into Aave v3;
+  - Optimising capital efficiency;
+- Migrating assets from Aave v2 to v3;
+  - Ensuring improved capital efficiency and enhanced risk management;
+
+Additionally, to further consolidate and mitigate risk, certain assets will be reallocated to the Core market from various Layer 2 instances. This adjustment aims to progressively consolidate liquidity into the latest instance of Aave, where it can continue to generate returns.
+
+To promote capital efficiency, some passive assets held idle are to be deployed into Aave to earn yield.
+
+Several smaller asset holdings are be converted to aEthETH to reduce the DAO’s exposure to thinly traded long tail assets (or products being sunset) to ensure a more resilient treasury structure.
+
+Finally, as part of the revamping of SKY, certain DAI-correlated assets will be swapped into aEthUSDS.
+
+## Specification
+
+### Transfer FLUID
+
+Transfer an initial 25% of the 1/3 FLUID holdings assigned to support GHO's growth and adoption on Fluid.
+
+Specifically, Transfer 95,417 $FLUID to Merit SAFE `0xdeadD8aB03075b7FBA81864202a2f59EE25B312b`
+
+Reference this prior [proposal](https://governance.aave.com/t/arfc-fluid-alignment-with-inst-purchase/19921) for more information.
+
+### Gas Reimbursement and EURA Bad Debt Clearing Reimbursement
+
+Transfer 26,500 USDC, 3.404 ETH and 149 PYUSD to ACI to reimburse expenses incurred.
+
+This reimburses associated expenses with clearing EURA bad debt, gas costs linked to AIP submissions and administering the PYUSD rewards program respectively.
+
+ACI Receiving Address: `0xac140648435d03f784879cd789130F22Ef588Fcd`
+
+### Migrate funds from Aave v2 to v3
+
+Migrate the following assets from Aave v2 to v3 on the respective network.
+
+Where it says All-1, it refers to all available liquidity minus 1 unit as to not drain the pool where possible. In other pools, it will be as much as is available to withdraw.
+
+|   Ethereum    |    Avalanche    |     Polygon     |
+| :-----------: | :-------------: | :-------------: |
+| aWETH (All-1) | avWETH (All-1)  | amWMATIC (All-1)|
+| aWBTC (All-1) |  avDAI (All-1)  | amUSDT (All-1)  |
+| aLINK (All-1) | avWAVAX (All-1) |                 |
+| aMKR (All-1)  |                 |                 |
+| aUSDC (1.5M)  |                 |                 |
+| aUSDT (2.0M)  |                 |                 |
+
+Funds are to be deposited into the Core deployment on Ethereum and Aave v3 on the respective networks.
+
+Note: Not all USDC and USDT on v2 Ethereum can be migrated due to high utilisation. This publication proposes a partial migration subject to liquidity availability.
```

Comment:
this should include the amounts specified in the payload contracts

Reply:
the amounts are listed in the table above. I could change the last part of the sentence to also include those values

Reply:
yes, i think it will be more clear this way